### PR TITLE
fix: Web 1992 token details update blocked token detail pages

### PR DESCRIPTION
**Pull request by lynnshaoyu**
https://uniswaplabs.atlassian.net/browse/WEB-1992?atlOrigin=eyJpIjoiM2Q5MGFjOWIwY2UzNGQ3M2IyMTE2NDFjNzE0YjAyNGMiLCJwIjoiaiJ9

### CODE REVIEW

File: src/components/Tokens/TokenDetails/index.tsx [hunk 1]
```diff
@@ -216,23 +218,27 @@ export default function TokenDetails({
...
         )}
 
-        <RightPanel>
-          <Widget
-            token={token ?? undefined}
-            defaultField={Field.OUTPUT}
-            onTokenChange={navigateToWidgetSelectedToken}
-            onReviewSwapClick={onReviewSwapClick}
-          />
+        <RightPanel onClick={() => isBlockedToken && setOpenTokenSafetyModal(true)}>
```
Comment (just-toby) [inline src/components/Tokens/TokenDetails/index.tsx:221]:
```suggestion
        <RightPanel onClick={isBlockedToken ? () => setOpenTokenSafetyModal(true) : null}>
```

? so we don't have an extra callback running unless needed?

**Review by just-toby — APPROVED** [review]
lgtm, one suggestion though. also, good candidate for a test ?

File: src/components/Tokens/TokenDetails/index.tsx (hunk 1 above)
Comment (lynnshaoyu) [reply src/components/Tokens/TokenDetails/index.tsx]: niceeee catch ty

Comment (lynnshaoyu) [reply src/components/Tokens/TokenDetails/index.tsx:221]: oops seems to lead to error actually / won't build

**Pull request merged**